datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -139,11 +147,15 @@ func (c *Consumer) Subscribe(ctx context.Context) error {
 
 	// Iterate over consumer sessions.
 	for {
+		Logger.Println("started to consume Kafka message")
```

Comment:
The log is not printed until the consumer has an error or is closed, right? Just to make sure it is not versboe.

Reply:
Right. This log will be printed whenever the session is restarted.